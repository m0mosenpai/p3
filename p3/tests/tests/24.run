../solution/wsh <tests/24.wsh
