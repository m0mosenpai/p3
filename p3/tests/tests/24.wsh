export class=cs537
local class=cs400
echo $class
