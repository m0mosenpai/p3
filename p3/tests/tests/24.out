wsh> wsh> wsh> cs537
wsh> 